tests/src/test/scala/cats/tests/XorTTests.scala

```diff
@@ -14,8 +14,6 @@ class XorTTests extends CatsSuite {
   implicit val iso = CartesianTests.Isomorphisms.invariant[XorT[List, String, ?]]
   checkAll("XorT[List, String, Int]", MonadErrorTests[XorT[List, String, ?], String].monadError[Int, Int, Int])
   checkAll("MonadError[XorT[List, ?, ?]]", SerializableTests.serializable(MonadError[XorT[List, String, ?], String]))
-  checkAll("XorT[List, String, Int]", MonoidKTests[XorT[List, String, ?]].monoidK[Int])
-  checkAll("MonoidK[XorT[List, String, ?]]", SerializableTests.serializable(MonoidK[XorT[List, String, ?]]))
```

Comment:
This is minor, but now that the `SemigroupK` instance no longer requires a `Semigroup[L]`, the `SemigroupK` tests down at line ~49 don't need the local `Semigroup[ListWrapper[String]]` instance and could be moved up here, similarly to [this comment](https://github.com/typelevel/cats/pull/996/files#r61065904).